src/Chapter1.hs

```diff
@@ -568,7 +576,8 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel c = elem c "aeiouy"
```

Comment:
Absolutely correct! 👍🏼 

One note in here, that sometimes, `elem` could be slower than the explicit pattern matching. I remember there were some benchmarks on one particular case, that showed how moving to pattern matching on each case separately drastically decrease time 🐎

Reply:
So the explicit pattern matching would be something like this?

```haskell
isVowel :: Char -> Bool
isVowel c
  | 'a' = True
  | 'e' = True
  ... all vowels
  | otherwise = False
```

Reply:
Yes, exactly!